selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/debugger/InnerClassAndLambdaDebuggingTest.java

```diff
@@ -140,7 +140,6 @@ public void shouldDebugMethodLocalInnerClass() {
 
     // then
     editor.waitActiveBreakpoint(53);
-    debugPanel.waitTextInVariablesPanel("methodValue=\"App method local inner test\"");
```

Comment:
Could you, please, explain, why did you remove this check? Does it mean that there will be no values inside the _Variables Panel_?

Reply:
Why did you decide that no variables will be inside the Variable Panel?
It means that `methodValue` is unavailable only.

Reply:
But **methodValue** variable should be there in _Variable Panel_ so as it is accessible from inside the **MethodLocalInner::getValue()** method and is returning by this method.
```javascript
    public String getAppValue() {
        final String methodValue = "App method local inner test";

        class MethodLocalInner {
            String getValue() {
                return methodValue;
            }
        }

        return new MethodLocalInner().getValue();
    }
```

Reply:
Sure, but the reference Eclipse IDE doesn't show that variable too. I think that intelejii does some trick to show that.

Reply:
You are right :-(
![screenshot from 2017-11-17 16-57-34](https://user-images.githubusercontent.com/1197777/32953326-87970f96-cbb8-11e7-8646-46d823392422.png)